src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/service/ChargeService.kt

```diff
@@ -35,10 +35,28 @@ class ChargeService(
   private val appearanceChargeHistoryRepository: AppearanceChargeHistoryRepository,
 ) {
 
-  private fun createChargeEntity(charge: CreateCharge, sentencesCreated: MutableMap<UUID, SentenceEntity>, prisonerId: String, courtCaseId: String, courtAppearanceId: String): RecordResponse<ChargeEntity> {
+  private fun createChargeEntity(
+    charge: CreateCharge,
+    sentencesCreated: MutableMap<UUID, SentenceEntity>,
+    prisonerId: String,
+    courtCaseId: String,
+    courtAppearanceId: String,
+  ): RecordResponse<ChargeEntity> {
+    var supersedingCharge: ChargeEntity? = null
```

Comment:
```suggestion
    val supersedingCharge: ChargeEntity? = charge.replacedChargeUuid?.let { chargeRepository.findFirstByChargeUuidAndStatusIdNotOrderByUpdatedAtDesc(it) }
```
if a kotlin way of writing this